packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -911,13 +920,20 @@ export class StripeHelper {
    *
    * @param customer
    */
-  hasOpenInvoice(customer: Stripe.Customer) {
-    const subscription = customer.subscriptions?.data.find(
-      (sub) =>
-        ACTIVE_SUBSCRIPTION_STATUSES.includes(sub.status) &&
-        (sub.latest_invoice as Stripe.Invoice).status === 'open'
+  async hasOpenInvoice(customer: Stripe.Customer) {
+    const activeInvoices = customer.subscriptions?.data
+      .filter((sub) => ACTIVE_SUBSCRIPTION_STATUSES.includes(sub.status))
+      .map((sub) => sub.latest_invoice)
+      .filter((invoice) => invoice !== null);
+    if (!activeInvoices) {
```

Comment:
I know this has already merged, but I had an in-progress comment here that I'll just ask anyway: if there are no active invoices, isn't `activeInvoices` an empty array? Isn't `[]` truthy?